articles/client-platforms/_includes/_rules-introduction.md

```diff
@@ -0,0 +1,33 @@
+Rules are one of the cool features of Auth0. Reason for it's coolness is it's flexibility which gives you the ability to extend what Auth0 has to offer. Rule are JavaScript functions which can be thought  of as middlewares because of how they work. See full docs [here](https://auth0.com/docs/rules)
```

Comment:
just a suggestion: "Rule are JavaScript functions which can be thought  of as middlewares because of how they work. See full docs [here](https://auth0.com/docs/rules)" -> "Auth0 rules are JavaScript functions which works like middlewares. To see a detailed description, please refer to [the full docs](https://auth0.com/docs/rules)